forage-android/src/main/java/com/joinforage/forage/android/core/Log.kt

```diff
@@ -55,6 +58,17 @@ internal interface Log {
                     .build()
             }
 
+            override fun initializeSentry(context: Context){
+                val flavor = BuildConfig.FLAVOR
+                if (flavor == "sandbox" || flavor == "prod"){
```

Comment:
Removed the check since we do not want to restrict environments to sandbox and prod. Unit testing would be under the ` getSilentInstance()` instance which does not initialise sentry